stream-chat-android-client/src/main/java/io/getstream/chat/android/client/socket/EventsParser.kt

```diff
@@ -44,8 +44,12 @@ internal class EventsParser(
     override fun onClosing(webSocket: WebSocket, code: Int, reason: String) { }
 
     override fun onClosed(webSocket: WebSocket, code: Int, reason: String) {
-        // Treat as failure and reconnect, socket shouldn't be closed by server
-        onFailure(ChatNetworkError.create(ChatErrorCode.SOCKET_CLOSED))
+        if (code == CODE_CLOSE_SOCKET_FROM_CLIENT) {
```

Comment:
If the connection was closed by the client, we doesn't need to forward an error.
On the other hand, because we closed it, we don't one to forward any new event we could receive from backend, that is because we update the flag